library/task-library/Sharepoint/Sharepoint_InstallPrereq.ps1

```diff
@@ -0,0 +1,54 @@
+##############################################
+# Name        : Sharepoint_InstallPrereq.ps1
+# Author      : Calm Devops
+# Version     : 1.0
+# Description : Script to download and start sharepoint prerequisites.
+# Compatibility : Windows2016
+##############################################
+
+cd $HOME
+$imageUrl="https://download.microsoft.com/download/0/0/4/004EE264-7043-45BF-99E3-3F74ECAE13E5/officeserver.img"
+$Outfile=$PWD.Path + "\officeserver.img"
+$wc = New-Object System.Net.WebClient
+$wc.DownloadFile($imageUrl, $Outfile)
+
+Mount-DiskImage -ImagePath  $Outfile 
+$driveLetter = (Get-DiskImage -ImagePath $Outfile | Get-Volume).DriveLetter 
+
+$driveLetter += ':'
+
+$SetupPath = $driveLetter + '\setup.exe'
+$PreReqPath = $driveLetter +'\prerequisiteinstaller.exe'
+
+Import-Module Servermanager
+
+$PreReqFiles = "C:\SharePoint_Prerequisites\PrerequisiteInstaller"
+mkdir $PreReqFiles
+
+$DownloadURL = "https://download.microsoft.com/download/F/3/C/F3C64941-22A0-47E9-BC9B-1A19B4CA3E88/ENU/x64/sqlncli.msi"
```

Comment:
Are the UUIDs anyway related to your login?

Reply:
Nope that's application related uuid. Click on that link would start download for you.